crates/topos-crypto/tests/messages.rs

```diff
@@ -20,7 +20,7 @@ pub fn test_signing_messages() {
         .expect("Cannot create Signature");
 
     let message_signer_receiver =
-        MessageSigner::new("a2e33a9bad88f7b7568228f51d5274c471a9217162d46f1533b6a290f0be1baf")
+        MessageSigner::new(b"a2e33a9bad88f7b7568228f51d5274c471a9217162d46f1533b6a290f0be1baf")
```

Comment:
This is not valid. `a2` should be byte 162, here Rust parses it as two bytes 97, 50
Is this really what you wanted https://doc.rust-lang.org/reference/tokens.html#byte-string-literals

Reply:
You can use instead:
```
hex::decode(""a2e33a9bad88f7b7568228f51d5274c471a9217162d46f1533b6a290f0be1baf").unwrap()
```